src/parameters/update_parameters.jl

```diff
@@ -543,10 +543,8 @@ function _update_parameter_values!(
             )
             variable_cost_forecast_values = TimeSeries.values(ts_vector)
             for (t, value) in enumerate(variable_cost_forecast_values)
-                if attributes.uses_compact_power
```

Comment:
This over writes the value variable in cases where compact_power is used. It is not properly tested but do not remove it

Reply:
That helper function `_convert_variable_cost` does not seem to be defined anywhere, so this code would just error if it ever came up. Is there a definition somewhere I should reinstate?

Reply:
Looks like this code hasn't been updated in 2 years. Julia will throw a not implemented method anyway. Leave the code and we will implement the method when we make the tests and they inevitable will fail. For now, we can put a TODO

Reply:
Done